neptune/new/internal/init/run.py

```diff
@@ -99,102 +99,107 @@ def init_run(
-            Make sure you are using the same identifier throughout the whole pipeline execution.
-        mode (str, optional): Connection mode in which the tracking will work. Defaults to `None`.
-            If `None`, the value of `NEPTUNE_MODE` environment variable will be taken.
-            If no value was set for environment variable then `'async'` at default.
-            Possible values 'async', 'sync', 'offline', 'read-only' and 'debug'.
-        name (str, optional): Editable name of the run. Defaults to `'Untitled'`.
-            Name is displayed in the run's Details and in Runs table as a column.
-        description (str, optional): Editable description of the run. Defaults to `''`.
-            Description is displayed in the run's Details and can be displayed in the runs view as a column.
-        tags (list of str or str, optional): Tags of the run. Defaults to `[]`.
-            They are editable after run is created.
-            Tags are displayed in the run's Details and can be viewed in Runs table view as a column.
-        source_files (list of str or str, optional): List of source files to be uploaded.
-            Uploaded sources are displayed in the run’s Source code tab.
-            Unix style pathname pattern expansion is supported. For example, you can pass '*.py' to upload all python
-            source files from the current directory.
-            If `None` is passed, Python file from which run was created will be uploaded.
-        capture_stdout (bool, optional): Whether to send run's stdout. Defaults to `True`.
-            Tracked metadata will be stored inside `monitoring_namespace`.
-        capture_stderr (bool, optional):  Whether to send run’s stderr. Defaults to `True`.
-            Tracked metadata will be stored inside `monitoring_namespace`.
-        capture_hardware_metrics (bool, optional): Whether to send hardware monitoring logs
-            (CPU, GPU, Memory utilization). Defaults to `True`.
-            Tracked metadata will be stored inside `monitoring_namespace`.
-        fail_on_exception (bool, optional): Whether to register an uncaught exception handler to this process and,
-            in case of an exception, set run's sys/failed to True. Exception is always logged
-        monitoring_namespace (str, optional): Namespace inside which all monitoring logs be stored.
+        project: Name of the project where the run should go, in the form "workspace-name/project_name".
+        If None (default), the value of the NEPTUNE_PROJECT environment variable is used.
+        api_token: User's API token. Defaults to None.
+            If None (default), the value of the NEPTUNE_API_TOKEN environment variable is used.
+            Note: To keep your API token secure, save it to the NEPTUNE_API_TOKEN environment variable rather than
+            placing it in plain text in the source code.
+        with_id: If you want to resume a run, the identifier of the existing run.
```

Comment:
Where is run param? It's still there so even if deprecated maybe we should have it with a proper info?

Reply:
You're right